rackcity/models/rack.py

```diff
@@ -31,7 +32,17 @@ class Meta:
                 name='unique rack letter and number per datacenter'),
         ]
 
+    def char_range(self, c1, c2):
+        """Generates the characters from `c1` to `c2`, inclusive."""
+        for c in range(ord(c1), ord(c2)+1):
+            yield chr(c)
```

Comment:
Is it possible to make this not take self as an argument so you can just call `char_range()` instead of `self.char_range()`?

Reply:
It would not let me unfortunately